guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -0,0 +1,163 @@
+<?php declare(strict_types=1);
+
+namespace Swag\ExamplePlugin\Core\Checkout\Customer\Event;
+
+**use Shopware\Core\Checkout\Customer\CustomerDefinition;
+use Shopware\Core\Checkout\Customer\CustomerEntity;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\CustomerAware;
+use Shopware\Core\Framework\Event\EventData\EntityType;
+use Shopware\Core\Framework\Event\EventData\EventDataCollection;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class ExampleEvent extends Event implements CustomerAware
+{
+    public const EVENT_NAME = 'example.event';
+
+    private CustomerEntity $customer;
+
+    private Context $context;
+
+    public function __construct(Context $context, CustomerEntity $customer)
+    {
+        $this->customer = $customer;
+        $this->context = $context;
+    }
+
+    public function getName(): string
+    {
+        return self::EVENT_NAME;
+    }
+
+    public function getCustomer(): CustomerEntity
+    {
+        return $this->customer;
+    }
+
+    public function getCustomerId(): string
+    {
+        return $this->customer->getId();
+    }
+
+    public static function getAvailableData(): EventDataCollection
+    {
+        return (new EventDataCollection())
+            ->add('customer', new EntityType(CustomerDefinition::class));
+    }
+
+    public function getContext(): Context
+    {
+        return $this->context;
+    }
+}
+```
+
+### Add your new event to the flow trigger list
+
+ At this step you need to add your new event to the flow trigger list, let see the code below:
+
+```php
+<?php declare(strict_types=1);
+
+namespace Swag\ExamplePlugin\Core\Checkout\Customer\Subscriber;
+
+use Shopware\Core\Framework\Event\BusinessEventCollector;
+use Shopware\Core\Framework\Event\BusinessEventCollectorEvent;
+use Swag\FlowBuilderProfessional\Core\Checkout\Customer\Event\ExampleEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class BusinessEventCollectorSubscriber implements EventSubscriberInterface
+{
+    private BusinessEventCollector $businessEventCollector;
+
+    public function __construct(BusinessEventCollector $businessEventCollector) {
+        $this->businessEventCollector = $businessEventCollector;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            BusinessEventCollectorEvent::NAME => ['onAddExampleEvent', 1000],
+        ];
+    }
+
+    public function onAddExampleEvent(BusinessEventCollectorEvent $event): void
+    {
+        $collection = $event->getCollection();
+
+        $definition = $this->businessEventCollector->define(ExampleEvent::class);
+
+        if (!$definition) {
+            return;
+        }
+
+        $collection->set($definition->getName(), $definition);
+    }
+}
+```
+
+Please note that you subscriber you has high priority to make sure your event is added before any subscriber`BusinessEventCollectorEvent` to prevent missing aware or action. I set 1000 for `onAddExampleEvent` action:
```

Comment:
There's one `you` too much in this sentence and a space ` ` missing before `BusinessEventCollectorEvent`.

> I set priority to 1000 for the `onAddExampleEvent` action: